client/store/cart.js

```diff
@@ -0,0 +1,90 @@
+import Axios from 'axios'
+
+const ADD_ITEM = 'ADD_ITEM'
+const GET_ITEMS = 'GET_ITEMS'
+const CLEAR_ITEMS = 'CLEAR_ITEMS'
```

Comment:
Not a functional comment but you can also do this as `const [ADD_ITEM, GET_ITEMS, CLEAR_ITEMS] = ["ADD_ITEM", "GET_ITEMS", "CLEAR_ITEMS"]`